docs/hoon/syntax.md

```diff
@@ -7,22 +7,19 @@ title: Syntax
 
 # Hoon syntax
 
-Hoon's semantics are simple, but unusual.  Hoon's syntax is
-complex, and unusual.  Sorry.
+Hoon's syntax is unusual, and to beginners it can look a lot more 
+complex than it really is. However, we believe that anyone who 
+learns Hoon syntax will find that it has a number of advantages.
```

Comment:
I think that this section might perhaps be a little bit too deferential. I'm inclined to keep the assertion that Hoon's semantics are simple. And its syntax really is complex.

Reply:
(To others: we're battling this out in urbit-meta now.)  I remain unconvinced.  The syntax is unusual, but it isn't as complex as it looks.  We don't want to demoralize newcomers who have heard rumors of Urbit's supposed inscrutability.

Reply:
Having read bits and pieces of the discussion in urbit-meta, I have to side with Joshua here.

Telling people "this is hard, good luck lol!" is setting them up for failure. Someone with little faith in their own ability to learn new things will jump ship _really_ hard at that. A little reassurance like "this is gonna be weird, but it's totally doable" goes a long way.

(Also, really, it's not so much the syntax that's complex, it's just that there's lots of runes to learn. I feel it might also be useful to point out that you only need to know a small subset of the runes to write working (albeit not very pretty) Hoon.)